scanner/config/config.go

```diff
@@ -42,41 +43,57 @@ var (
 		MTLS: MTLSConfig{
 			CertsDir: "",
 		},
+		Proxy: ProxyConfig{
+			ConfigFile: "config.yaml",
+		},
 		LogLevel: LogLevel(zerolog.InfoLevel),
 	}
 )
 
 // Config represents the Scanner configuration parameters.
 type Config struct {
-	Indexer        IndexerConfig `yaml:"indexer"`
-	Matcher        MatcherConfig `yaml:"matcher"`
-	HTTPListenAddr string        `yaml:"http_listen_addr"`
-	GRPCListenAddr string        `yaml:"grpc_listen_addr"`
-	MTLS           MTLSConfig    `yaml:"mtls"`
-	LogLevel       LogLevel      `yaml:"log_level"`
+	StackRoxServices bool          `yaml:"stackrox_services"`
```

Comment:
nit: I had to look at the implementation to figure out what `StackRoxServices` was doing.  It appears that when it's `true` traffic is allowed from indexer/matcher to Central + Sensor. Can this be renamed to something more descriptive? first thought: `AllowInternalStackroxTraffic`

Reply:
This configuration is meant to symbolize the scanner is deployed alongside StackRox services. This is used internally to determine the HTTP transports, but I think this meaning make it extendable to more internal conditionals, if needed. This is documented in the IndexerConfig and MatcherConfig, but I can add it here, too, to make it clearer here